src/API/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "PostgresConnection": "Host=db;Database=employeedb;Username=postgres;Password=postgres"
```

Comment:
Connection string do bazy w appsetings to nie jest idealne rozwiązanie, warto by użyć sekretów/vaulta.  Lecz na potrzeby zadania rekrutacyjnego jest to akceptowalne :)